### PR TITLE
Add set RUST_BACKTRACE=1 when using powershell on windows

### PR DESCRIPTION
**Pull request by Giuliano1993**
#### What kind of changes does this PR include?
- New or updated content

#### Description
- What does this PR change? Give us a brief description. 
The provided command for setting RUST_BACKTRACE=1 on Windows was valid only if using cmd; while, if using PowerShell, that  command wouldn't work.

Added the powershell version too

### CODE REVIEW

File: docs/guides/debugging/application.md [hunk 1]
```diff
@@ -18,13 +18,21 @@ Sometimes you may have an error in your Rust code, and the Rust compiler can giv
 RUST_BACKTRACE=1 tauri dev
 ```
 
-or like this on Windows:
+or like this on Windows if you're on cmd:
```
Comment (FabianLars) [inline docs/guides/debugging/application.md]:
```suggestion
or like this on Windows if you're using cmd.exe:
``` adding the .exe makes it a bit clearer.

File: docs/guides/debugging/application.md [hunk 2]
```diff
@@ -18,13 +18,21 @@ Sometimes you may have an error in your Rust code, and the Rust compiler can giv
...
 
 ```shell
 set RUST_BACKTRACE=1
 tauri dev
 ```
 
+or like this on Windows if you're using PowerShell:
+
+```shell
```
Comment (FabianLars) [inline docs/guides/debugging/application.md]: i think you can use `powershell` instead of `shell` here. should fix the highlighting.

**Review by FabianLars — CHANGES_REQUESTED** [review]
thank you! just 2 small things :)

File: docs/guides/debugging/application.md (hunk 2 above)
Comment (Giuliano1993) [reply docs/guides/debugging/application.md]: Yes you're right! Fixed it right away!

**Comment by Giuliano1993** [comment]
Thank you for the review @FabianLars ! Now it should be okay! :)

**Review by FabianLars — APPROVED** [review]
thanks!

**Pull request merged**